plugin.video.otaku/resources/lib/pages/nyaa.py

```diff
@@ -63,8 +63,8 @@ def _process_nyaa_episodes(self, url, episode, season=None):
             for i in soup.select("tr.danger,tr.default,tr.success")
         ]
 
-        regex = r'\ss(\d+)|season\s(\d+)|(\d+)+(?:st|[nr]d|th)\sseason'
-        regex_ep = r'\de(\d+)\b|\se(\d+)\b|\s-\s(\d{1,3})\b'
+        regex = r'\b(?:s|season|series)\s*(\d+)\b(?:episode|ep|eps)\s*(\d+)'
+        regex_ep = r'\b(?:e|ep|eps|episode)\s*(\d{1,3})\b'
```

Comment:
Please change it to `\d{1,3}` change it to `\d{1,4}` to handle long anime such One Piece and Detective Conan

Reply:
i was looking at that also thanks for pointing that out. appreciate ya a ton